### PR TITLE
Fix: Don't cache Mixlib::ShellOut

### LINKED ISSUE #15: Don't cache the Mixlib::ShellOut instance
We're caching [the Mixlib::ShellOut instance](https://github.com/rapid7/cassandra-utils/blob/0.3.1-beta.1/lib/cassandra/utils/cli/base.rb#L18) when running commands. This has bitten us before (see [PR #33 in daemon_runner](https://github.com/rapid7/daemon_runner/pull/33)). We should either skip the caching or just change this to use daemon_runner.

### PR DESCRIPTION
**Pull request by fmitchell-r7**
This fixes #15 by switching `Mixlib::ShellOut` to `DaemonRunner::ShellOut`. The [`daemon_runner` gem\(https://github.com/rapid7/daemon_runner) does the right thing and doesn't cache the `Mixlib::ShellOut` instance between executions.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -10,16 +10,16 @@ GIT
 PATH
   remote: .
   specs:
-    cassandra-utils (0.2.4)
-      daemon_runner (~> 0.2)
+    cassandra-utils (0.0.1)
```
Comment (rdickey-r7) [inline Gemfile.lock]: Was this change intentional?

Comment (fmitchell-r7) [inline Gemfile.lock]: I don't have a local VERSION file, so it reverted to the 0.0.1 version. As far as I know, this shouldn't break anything. @athompson-r7, any thoughts on this?

Comment (fmitchell-r7) [inline Gemfile.lock]: @rdickey-r7, we're using [thor-scmversion](https://github.com/RiotGamesMinions/thor-scmversion) to handle versioning in the CI job. It reads the version from tags in the Git repo, so the version in Gemfile.lock is ignored.

Comment (rdickey-r7) [inline Gemfile.lock]: Cool, good to know.
